minor structure changes re ElasticSearch (#64)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc

[id='deploying-stf-to-the-openshift-environment_{context}']
= Deploying {ProjectShort} to the {OpenShiftShort} environment

There are two ways in which you can deploy {ProjectShort} to the {OpenShiftShort} environment:

> * With ElasticSearch. For more information, see <<deploying-stf-to-the-openshift-environment-with-elasticsearch>>.

Reviewer: ```suggestion
* Storage of Events with ElasticSearch. For more information, see <<deploying-stf-to-the-openshift-environment-with-elasticsearch>>.
```

> * Without ElasticSearch. For more information, see <<deploying-stf-to-the-openshift-environment-without-elasticsearch>>.

Reviewer: ```suggestion
* Without ElasticSearch and disabling Events support. For more information, see <<deploying-stf-to-the-openshift-environment-without-elasticsearch>>.
```

[id='subscribing-to-the-amq-certificate-manager-operator']
== Subscribing to the AMQ Certificate Manager Operator

> You must subscribe to the AMQ Certificate Manager Operator prior to the deployment of the other components because the AMQ Certificate Manager Operator runs in a global namespace and is not compatible with other namespace scoping operators.

Reviewer: ```suggestion
You must subscribe to the AMQ Certificate Manager Operator prior to the deployment of the other components because the AMQ Certificate Manager Operator runs in a global namespace and is not compatible with the dependency management of Operator Lifecycle Manager when used with other namespace scoped operators.
```

> You must subscribe to the AMQ Certificate Manager Operator prior to the deployment of the other components because the AMQ Certificate Manager Operator runs in a global namespace and is not compatible with the dependency management of Operator Lifecycle Manager when used with other namespace scoped operators.

Reviewer: ```suggestion
You must subscribe to the AMQ Certificate Manager Operator prior to the deployment of the other components because the AMQ Certificate Manager Operator runs globally-scoped and is not compatible with the dependency management of Operator Lifecycle Manager when used with other namespace-scoped operators.
```

Reviewer: Re-read and made another tweak here to the wording

[id='subscribing-to-elastic-cloud-on-kubernetes-operator']
== Subscribing to the Elastic Cloud on Kubernetes Operator

> Before you install the Service Telemetry Operator and if you plan to store events in to ES, you need to enable the Elastic Cloud Kubernetes Operator.

Reviewer: ```suggestion
Before you install the Service Telemetry Operator and if you plan to store events in to ElasticSearch, you need to enable the Elastic Cloud Kubernetes Operator.
```

Reviewer: We call everything else out in this sentence, feels weird to have only one acronym. I didn't notice a line with _ElasticSearch (ES)_ either. Personally I would prefer to just call out _ElasticSearch_ every time.

Author: Fair point. Also, I don't think this acronym exists upstream.

[id='subscribing-to-the-service-telemetry-operator']
== Subscribing to the Service Telemetry Operator

> To instantiate an {ProjectShort} instance, create the ST object to allow the ST operator to create the environment.

Reviewer: ```suggestion
To instantiate an {ProjectShort} instance, create the `ServiceTelemetry` object to allow the Service Telemetry Operator to create the environment.
```

Reviewer: We never say _ST operator_ so I don't want to do that here. We can either use _STO_ or just always call out _Service Telemetry Operator_

[id='creating-a-servicetelemetry-object-in-openshift']
== Creating a ServiceTelemetry object in {OpenShiftShort}

> To deploy the Service Telemetry Framework, you must create an instance of ServiceTelemetry in {OpenShiftShort}. By default, `eventsEnabled` is set to false. If you do not want to store events in ElasticSearch, ensure that `eventsEnabled` is set to false. For more information, see <<deploying-stf-to-the-openshift-environment-without-elasticsearch>>.

Reviewer: ```suggestion
To deploy the Service Telemetry Framework, you must create an instance of `ServiceTelemetry` in {OpenShiftShort}. By default, `eventsEnabled` is set to false. If you do not want to store events in ElasticSearch, ensure that `eventsEnabled` is set to false. For more information, see <<deploying-stf-to-the-openshift-environment-without-elasticsearch>>.
```

[discrete]
=== Procedure

> . To store events in ElasticSearch, set `eventsEnabled` to true:

Reviewer 2: ```suggestion
. To store events in ElasticSearch, set `eventsEnabled` to true during deployment:
```
